src/_data/resources.json

```diff
@@ -500,6 +500,12 @@
 			"additional": "Microsoft",
 			"url": "https://docs.microsoft.com/en-us/learn/paths/accessibility-fundamentals/"
 		},
+		{
+			"title": "An e-learning platform on Web Accessibility",
```

Comment:
Can we change this to `The A11Y Collective`? We use the name of the resource be the title.